js/app.js

```diff
@@ -241,9 +244,9 @@ function setProductState (productObject, currentImages, previousImages, userClic
   };
   var stringifiedProductState = JSON.stringify(productState);
   localStorage.setItem('productState',stringifiedProductState);
-  storageProductState = localStorage.getItem('productState');
-  var parsedProductState = JSON.parse(storageProductState);
-  return parsedProductState;
```

Comment:
you're not using the return value from setProductState anywhere but you should still toss a return; at the end of the function